src/plugins/flare/flareplugin.cpp

```diff
@@ -333,7 +338,18 @@ bool FlarePlugin::write(const Tiled::Map *map, const QString &fileName)
                     out << ",";
                 out << "\n";
             }
-            out << "\n";
+            //Write all properties for this layer
+			Properties::const_iterator it = tileLayer->properties().constBegin();
+			Properties::const_iterator it_end = tileLayer->properties().constEnd();
+			for (; it != it_end; ++it) {
+                if(strcmp(it->typeName(), "Tiled::FilePath") == 0){
```

Comment:
Rather than doing a string comparison, please use `it->userType() == filePathTypeId()`.